src/main/java/org/opentripplanner/routing/core/StateEditor.java

```diff
@@ -235,9 +246,9 @@ public void incrementWeight(double weight) {
      * time is inferred from the direction of traversal. This is the only element of state that runs
      * backward when traversing backward.
      */
-    public void incrementTimeInSeconds(int seconds) {
+    public void incrementTimeInSeconds(int seconds, boolean beginningVehicleRenting) {
```

Comment:
Prawie wszędzie ta metoda jest wywoływana  z ``beginningVehicleRenting`` ustawionym na false.  Wygodniej byłoby dać to jako domyślną wartość i pominąć dodanie tego false'a w  30 plikach

Reply:
Słuszna uwaga - zrobione :)